marklogic-data-hub/src/test/java/com/marklogic/hub/dhs/DeployUserAmpsTest.java

```diff
@@ -0,0 +1,185 @@
+package com.marklogic.hub.dhs;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.document.DocumentManager;
+import com.marklogic.client.document.DocumentWriteSet;
+import com.marklogic.client.eval.ServerEvaluationCall;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.Format;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.HubClient;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.mgmt.ManageClient;
+import com.marklogic.mgmt.ManageConfig;
+import com.marklogic.mgmt.resource.security.AmpManager;
+import com.marklogic.mgmt.resource.security.RoleManager;
+import com.marklogic.mgmt.resource.security.UserManager;
+import com.marklogic.rest.util.ResourcesFragment;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static com.marklogic.client.io.DocumentMetadataHandle.Capability.READ;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class DeployUserAmpsTest extends AbstractHubCoreTest {
+    ObjectMapper mapper = new ObjectMapper();
+
+    @BeforeEach
+    void checkIfTestsCanBeRun(){
+        Versions.MarkLogicVersion mlVersion = versions.getMLVersion();
+        assertTrue(mlVersion.isNightly() || (mlVersion.getMajor() > 10) ||(mlVersion.getMajor() == 10 && mlVersion.getMinor() >= 440));
+    }
+
+    @Test
+    void createAmpWithInheritableRole() {
+        HubClient operatorHubClient = runAsDataHubOperator().newHubClient();
+        HubClient adminHubClient = runAsAdmin().newHubClient();
```

Comment:
Pinging @fsnow @ryanjdew  @rahulvudutala  @akshaysonvane as an FYI - I really like this - I think this is the pattern we eventually want for our tests, which is that we obtain upfront the necessary HubClients and give them descriptive names (I think they can be shortened to operatorClient and adminClient too, but that's beside the point). This avoids wondering - what is the current state of adminHubConfig in this test? Essentially, we want to stop using adminHubConfig except to a) get instances of HubClient, and b) to access HubProject.